config/gen.go

```diff
@@ -0,0 +1,45 @@
+//go:build ignore
+// +build ignore
+
+// This is used with `go generate` to copy schema.json into schema.go.
```

Comment:
Forgot to `git add` this earlier. It just copies over the schema.json into schema.go, so that we can edit the schema.json directly, use it with other schema processing tools potentially, and etc.

If we change the schema.json, we need to run `go generate ./...` to update schema.go, and then commit the file.

I haven't added a CI check to validate the schema.go is up to date, yet. I left that for a follow up.

Reply:
see my comment about go:embed

Reply:
Yep, go:embed is perfect for this. Since `go build` handles go:embed, the schema content is committed in only one place (schema.json), no custom script, no `go generate` or any extra build commands, and no CI check needed to ensure the schema.go is up to date. 🎉